app/services/mailer/transactional_email.rb

```diff
@@ -0,0 +1,76 @@
+require "mandrill"
+
+class TransactionalEmail
+  attr_reader :mandrill, :template_name, :brand, :merge_params, :message
+
+  module Types
+    WELCOME     = 'welcome'
+    PLAN_CHANGE = 'plan-change'
+    CANCEL_PLAN = 'cancel-plan'
+  end
+
+  def initialize(template_name, brand)
+    @mandrill = Mandrill::API.new(ENV["MANDRILL_API_KEY"])
+    @template_name = template_name
+    @brand = brand
+    @to = brand.twitter_admin.email
+    @email_attributs = get_email_attributes
```

Comment:
We don't need to set this here since `get_email_attributes` is an internal method. Let's rename the method to `email_attributes` and avoid setting the instance variable.